app/src/androidTest/java/com/android/sample/ui/endtoend/ActivityFlowTest.kt

```diff
@@ -164,31 +215,55 @@ class ActivityFlowTest {
   }
 
   @Test
-  fun aUserSignsInAndLooksAtTheirProfile() {
+  fun aUserSwitchesProfile() {
     // use of !! is allowed because this is a test environment and we know the
     // key exists. if it doesn't, it is up to the developer to fix the test
-    val email = defaultUserCredentials["email"]!!
-    val password = defaultUserCredentials["password"]!!
-    val name = defaultUserCredentials["first name"]!! // @TODO: Should this change to full name?
+    val email1 = defaultUserCredentials1["email"]!!
+    val password1 = defaultUserCredentials1["password"]!!
+    val name1 = defaultUserCredentials1["first name"]!! // @TODO: Should this change to full name?
+    val email2 = defaultUserCredentials1["email"]!!
+    val password2 = defaultUserCredentials1["password"]!!
+    val name2 = defaultUserCredentials1["first name"]!!
```

Comment:
Careful ! Here you use the same credentials, which is not what we are expected to do !! Will request changes !